aldjemy/orm.py

```diff
@@ -91,7 +91,13 @@ def _extract_model_attrs(metadata, model, sa_models):
                 secondary=sec_table,
                 primaryjoin=(sec_table.c[sec_column] == table.c[model_pk]),
                 secondaryjoin=(sec_table.c[p_sec_column] == p_table.c[p_name]),
-                overlaps=",".join([fk.m2m_field_name(), fk.m2m_reverse_field_name()]),
+                overlaps=",".join(
+                    (
+                        fk.m2m_field_name(),
+                        fk.m2m_reverse_field_name(),
+                        fk.remote_field.field.get_attname(),
+                    )
+                ),
```

Comment:
I was wondering what I should be thinking about this. This was definitely beyond my understanding of sqlalchemy to know what to do here. Thank you!

Reply:
The warning message was carefully crafted to explain what are the multiple possibilities to solve it. By elimination, it seems that, this solutiuon is the most suitable for us (to be as close as possible as django orm semantics).